data/map planets.txt

```diff
@@ -4958,3 +4958,67 @@ planet Zydee
+	shipyard "Advanced Southern Pirates"
+	shipyard "Betelgeuse Advanced"
+	shipyard "Betelgeuse Basics"
+	shipyard "Southern Pirates"
+	outfitter "Ammo South"
+	outfitter "Basic Outfits"
+	outfitter "Basic Weapons"
+	outfitter "Common Outfits"
+	outfitter "Pirate Outfits"
+	security 0.5
+
+planet "Mendez Station"
+	attributes CCOR station
+	landscape land/station20
+#	description "Mendez Station is the closest object to Fumalasamakah. Located in a low orbit just above the stellar corona, the monumentous ram-scoops built into the star-facing side of the station collect stellar ejections to process into fuel. The station is perpetually hot, even in the habitation sections as an inescapable effect of begin so close to a great mass of plasma undergoing nuclear fusion, however it would surely be much more so if it weren't for the primarily polyfiber construction and heat-radiation spires built into the far side of the station."
+#	spaceport "Rather than a traditional docking ring, part of which would be facing Fumalasamakah and its incredible heat and radiation at all times Mendez's 'spaceport' is actually a separate station located in a significantly higher stellar orbit. The station serves primarily as a secure dock for more normal ships while their inhabitants are taken to Mendez station via specialized transports built to resist the intense heat."
+	bribe 1
+	security 0.33
+
+planet Jakobsen
+	attributes CCOR farming
+	landscape land/sea4
+	description "Jakobsen is an ocean world, having approximately 6% of its surface above sea level. The land is almost exclusively volcanic islands and atolls as any land not refreshed by plate tectonics and slowly being eroded away by the immense seasonal storms that wreathe the planet. Despite these conditions, Jakobsen is an agricultural world, with vast fields of algae cultivated both to sustain the atmosphere's oxygen content and provide a nutritious, if bland, staple food."
+	spaceport "The spaceport is a wholly artificial island constructed near the center of one of the larger archipelagos. Surrounded by wave-breakers and raised several meters above the water line, the landing platforms are partially protected from the dangers of the planet's storms, additionally the smaller pads are able to be lowered into hangars so as to not be swept away during storms."
+	outfitter "Basic Outfits"
+	outfitter "Common Outfits"
+	bribe 1
+	security 0.33
+
+planet "Alvorada"
+	attributes CCOR military station
+#	description "Fortaleza da Alvorada Station is a grand fortress in space, built into the inner reaches of a giant asteroid surrounded by a field of smaller, yet still sizable, space-rocks. Serving as the seat of the CCOR government the station houses the offices of the members of the politburo, their meeting chambers, and grand server-banks to store and be able to process all information about the nation needed to make planning decisions."
```

Comment:
"Serving as the seat of the CCOR government, the station houses the offices of the members of the politburo and their meeting chambers- as well as grand server banks to store and process..."

Reply:
FYI the CCOR dissolved prior to the start of Endless Sky, and these are all placeholders from when the CCOR was meant to still exist in-game.
<img width="786" alt="Screenshot 2023-12-10 at 12 35 36 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/2506a803-b327-4e34-aae8-2da394933c59">
You can use this button (highlighted) to create suggestions. Edit the line that's been selected by editing the text under `suggestion`.